prov/usnic/src/usdf_cq.c

```diff
@@ -458,6 +463,11 @@ usdf_cq_post_soft(struct usdf_cq_hard *hcq, void *context, size_t len,
 	}
 
 	cq->c.soft.cq_last_op = USDF_SOFT_CQ_WRITE;
+
+	/* TODO: How do I report this if it fails?!?!? */
```

Comment:
Should retry if `EINTR`.  The other cases could be covered by an `assert`, since this really shouldn't fail otherwise.